snuba/migrations/runner.py

```diff
@@ -494,6 +555,34 @@ def get_status(migration_key: MigrationKey) -> Status:
 
         return group_migrations
 
+    def _get_completed_migrations(
+        self, groups: Sequence[MigrationGroup]
+    ) -> List[MigrationKey]:
+        """
+        Get a list of completed migrations for a list of groups
+        """
+        migration_status = self._get_migration_status()
+
+        def get_status(migration_key: MigrationKey) -> Status:
+            return migration_status.get(migration_key, Status.NOT_STARTED)
```

Comment:
Not sure why this function is needed.  Why don't we just do:
```
migration_status.get(migration_key, Status.NOT_STARTED)
```
instead of adding this function call overhead?